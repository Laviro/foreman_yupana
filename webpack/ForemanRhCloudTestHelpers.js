export const inventoryStateWrapper = innerState => ({
  ForemanRhCloud: {
    inventoryUpload: { ...innerState },
  },
});
