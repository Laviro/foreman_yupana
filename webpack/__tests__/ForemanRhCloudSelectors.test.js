import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
import { inventoryStateWrapper } from '../ForemanRhCloudTestHelpers';
import {
  selectForemanRhCloud,
  selectForemanInventoryUpload,
  selectInsightsCloudSync,
} from '../ForemanRhCloudSelectors';

const state = rhCloudStateWrapper({ inventoryReducersNamespaces: {} });

const fixtures = {
  'should return ForemanRhCloud': () => selectForemanRhCloud(state),
  'should return ForemanInventoryUpload': () =>
    selectForemanInventoryUpload(state),
  'should return InsightsCloudSync': () => selectInsightsCloudSync(state),
};

describe('ForemanRhCloud selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
