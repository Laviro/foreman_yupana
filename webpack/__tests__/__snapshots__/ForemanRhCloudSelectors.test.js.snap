// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanRhCloud selectors should return ForemanInventoryUpload 1`] = `
Object {
  "inventoryReducersNamespaces": Object {},
}
`;

exports[`ForemanRhCloud selectors should return ForemanRhCloud 1`] = `
Object {
  "inventoryUpload": Object {
    "inventoryReducersNamespaces": Object {},
  },
}
`;
